layer/nrlf/core/tests/test_pydantic_errors.py

```diff
@@ -85,42 +85,11 @@ def test_validate_content_missing_format():
                 }
             ]
         },
-        "diagnostics": "Failed to parse DocumentReference resource (content[0].format: Field required. See ValueSet: https://fhir.nhs.uk/England/CodeSystem/England-NRLFormatCode)",
+        "diagnostics": "Failed to parse DocumentReference resource (content[0].format: Field required)",
```

Comment:
Double check we cannot use this

Reply:
The only way to do this is to add specific code to the general `errors.py` class or do the pydantic checks at a later point. I won't do any of those as the code is cleaner as it is and there is little value on adding this extra information.